__tests__/app.test.js

```diff
@@ -257,6 +257,30 @@ describe("PATCH /api/articles/:article_id", () => {
     })
 })
 
+describe("DELETE /api/comments/comment_id", () => {
+    test("204 - deletes the specified comment and sends no body back", () => {
+        return request(app)
+            .delete("/api/comments/17")
+            .expect(204)
```

Comment:
Your test description says that no body should be sent back, but you haven't tested for this 👀